docs/docs/timelock-encryption.md

```diff
@@ -0,0 +1,119 @@
+That said, `fastnet` (our new network for timelock encryption) was started with 16 different participants running 23 nodes, so we're confident a threshold number of them have not been compromised!
+
+- **neither the scheme, nor drand are quantum resistant**
+
+The cryptography we employ do not use quantum resistant algorithms. If you timelock encrypt something for 1000 years and a viable quantum computer becomes available, anybody can decrypt it. Right now there are no widespread quantum resistant schemes for threshold cryptography, but we're keeping an eye out for the future!
+To that end, consider it relatively safe to timelock encrypt something for 5-20 years depending on how bullish you are on quantum computing.
+
+- **if the League of Entropy shuts down, members will delete their keys**
+
+The world is unpredictable (just like drand randomness... heh), and it's possible that the League of Entropy will all hang up their coats at some point in the future. Should that happen, members would have two choices with their private keys: release them to the world or delete them entirely.
+The former would mean that ciphertexts created for some time after the cessation of the network would be decryptable to everybody. The latter would mean that ciphertexts created for some time after the cessation of the network would be unencryptable forever (/until quantum computers can break them).
+In the interests of privacy, we felt the latter option was preferable. That said, if you encrypt the private key to your Filecoin fortune to stop yourself from spending it now and the network stops... you're going to have a bad time.
+
+[1] their report is available in timelocked form [here](https://research.kudelskisecurity.com/2023/01/09/announcing-a-timelocked-responsible-disclosure/), although it was encrypted using the drand testnet, so you will need to use the [tlock go library](https://github.com/drand/tlock) to decrypt it by passing in the testnet details. All the security issues they identified have since been remediated! You can find remediation details in the github issues of the relevant repos.
+
+## Use cases
+
+We have identified some cool use cases for timelock encryption, but these are certainly not exhaustive - hopefully you can come up with new, innovative use cases we haven't thought of!
+
+- [responsible vulnerability disclosure](https://timevault.drand.love/)
+- transfer of assets/password upon death
+- sealed bid auctions
+- [MEV prevention](https://coinmarketcap.com/alexandria/glossary/miner-extractable-value-mev)
+- public turn-based games
+- voting
+- quizzes
+
+## The math bit 🤓
+
+### Encryption
+
+A client that wishes to encrypt a message $M \in {0,1}^l$ only decryptable at the epoch $p$ will perform the following:
+
+1. Compute $\mathbb{G_{id}}= e(P, Q_{id}) = e(P,H_1(p))$, the "round public key" 
+- (this can be pre-computed per epoch, it's the same for everyone)
+
```

Comment:
```suggestion
```

Reply:
Not sure how it's rendering but usually lists in MD are not separated with new lines as it would mean a new list each time with a single item